### PR TITLE
translate gremlin console to English

### PR DESCRIPTION
**Pull request by JackyYangPassion**
impelement https://github.com/apache/incubator-hugegraph-doc/issues/109
translate gremlin-console.md doc

### CODE REVIEW

File: content/en/docs/clients/gremlin-console.md [hunk 1]
```diff
@@ -260,9 +260,9 @@ schema = hugegraph.schema()
 g = hugegraph.traversal()
 ```
 
-这样在console中便可以直接使用schema和g这两个对象，做元数据的管理和图的查询了。
+In this way, the two objects schema and g can be directly used in the console for metadata management and graph query.
```
Comment (javeme) [inline content/en/docs/clients/gremlin-console.md:263]: can wrap schema and g with \``

File: content/en/docs/clients/gremlin-console.md [hunk 2]
```diff
@@ -212,7 +212,7 @@ plugin activated: tinkerpop.utilities
 plugin activated: tinkerpop.tinkergraph
 ```
 
-连接server，需在配置文件中指定连接参数，在conf目录下有一个默认的remote.yaml
+To connect to the server, you need to specify the connection parameters in the configuration file, and there is a default remote.yaml in the conf directory
```
Comment (javeme) [inline content/en/docs/clients/gremlin-console.md:215]: a default remote.yaml file

File: content/en/docs/clients/gremlin-console.md [hunk 3]
```diff
@@ -71,11 +71,11 @@ System.out.println(">>>> query all vertices: size=" + g.V().toList().size());
 System.out.println(">>>> query all edges: size=" + g.E().toList().size());
 ```
 
-其实这一段groovy脚本几乎就是Java代码，不同之处仅在于变量的定义可以不写类型声明，以及每一行末尾的分号可以去掉。
+In fact, this groovy script is almost Java code, the only difference is that the variable definition can be written without the type declaration, and the semicolon at the end of each line can be removed.
 
-> g.V() 是获取所有的顶点，g.E() 是获取所有的边，toList() 是把结果存到一个 List 中，参考[TinkerPop Terminal Steps](http://tinkerpop.apache.org/docs/current/reference/#terminal-steps)。
+> g.V() is to get all the vertices, g.E() is to get all the edges, toList() is to store the result in a List, refer to[TinkerPop Terminal Steps](http://tinkerpop.apache.org/docs/current/reference/#terminal-steps)。
```
Comment (javeme) [inline content/en/docs/clients/gremlin-console.md:76]: can wrap g.V()/g.E() with \``

File: content/en/docs/clients/gremlin-console.md [hunk 4]
```diff
@@ -114,13 +114,13 @@ gremlin> g.E()
...
+For more Gremlin statements, please refer to [Tinkerpop Official Website](http://tinkerpop.apache.org/docs/current/reference/)
 
-### 2 Client/Server请求模式
+### 2 Client/Server request mode
 
-因为Gremlin-Console只能通过WebSocket连接HugeGraph-Server，默认HugeGraph-Server是对外提供HTTP连接的，所以先修改gremlin-server的配置。
+Because Gremlin-Console can only connect to HugeGraph-Server through WebSocket, HugeGraph-Server provides HTTP connections by default, so modify the configuration of gremlin-server first.
 
-*注意：将连接方式修改为WebSocket后，HugeGraph-Client、HugeGraph-Loader、HugeGraph-Studio等配套工具都不能使用了。*
+*Note: After changing the connection method to WebSocket, HugeGraph-Client, HugeGraph-Loader, HugeGraph-Studio and other supporting tools cannot be used.*
```
Comment (javeme) [inline content/en/docs/clients/gremlin-console.md:123]: TODO: now we support HttpAndWebSocketChannelizer, we need to update this doc later

File: content/en/docs/clients/gremlin-console.md [hunk 5]
```diff
@@ -4,18 +4,18 @@ linkTitle: "Gremlin Console"
 weight: 3
 ---
 
-Gremlin-Console是由Tinkerpop自己开发的一个交互式客户端，用户可以使用该客户端对Graph做各种操作，主要有两种使用模式：
+Gremlin-Console is an interactive client developed by Tinkerpop. Users can use this client to perform various operations on Graph. There are two main usage modes:
```
Comment (javeme) [inline content/en/docs/clients/gremlin-console.md:7]: Tinkerpop => TinkerPop

File: content/en/docs/clients/gremlin-console.md [hunk 6]
```diff
@@ -4,18 +4,18 @@ linkTitle: "Gremlin Console"
...
-- 单机离线调用模式；
-- Client/Server请求模式；
+- Stand-alone offline call mode;
+- Client/Server request mode;
 
-### 1 单机离线调用模式
+### 1 Stand-alone offline call mode
 
-由于lib目录下已经包含了HugeCore的jar包，且HugeGraph已经作为插件注册到Console中，用户可以直接写groovy脚本调用HugeGraph-Core的代码，然后交由Gremlin-Console内的解析引擎执行，就能在不启动Server的情况下操作图。
+Since the lib directory already contains the HugeCore jar package, and HugeGraph has been registered in the Console as a plug-in, the users can write a groovy script directly to call the code of HugeGraph-Core, and then hand it over to the parsing engine in Gremlin-Console for execution. As a result, the users  can operate the graph without starting the Server.
```
Comment (javeme) [inline content/en/docs/clients/gremlin-console.md:14]: there are 2 spaces in "users can"

File: content/en/docs/clients/gremlin-console.md [hunk 7]
```diff
@@ -4,18 +4,18 @@ linkTitle: "Gremlin Console"
 weight: 3
 ---
 
-Gremlin-Console是由Tinkerpop自己开发的一个交互式客户端，用户可以使用该客户端对Graph做各种操作，主要有两种使用模式：
+Gremlin-Console is an interactive client developed by Tinkerpop. Users can use this client to perform various operations on Graph. There are two main usage modes:
 
-- 单机离线调用模式；
-- Client/Server请求模式；
+- Stand-alone offline call mode;
```
Comment (javeme) [inline content/en/docs/clients/gremlin-console.md:9]: "Stand-alone offline mode" is ok

**Review by javeme — COMMENTED** [review]
Thanks for the translation, some tiny comments

**Comment by JackyYangPassion** [comment]
> Thanks for the translation, some tiny comments

Thanks for your review, I will fix it asap

**Review by imbajin — APPROVED** [review]

**Pull request merged**